### DISCUSSION TITLE
Multiple Process not Launch as Expected in Flask-Scoketio

### DISCUSSION THREAD

**Discussion by zaingithub** [miguelgrinberg/Flask-SocketIO#1818] [Q&A]
Hi, I'm using flask-socketio to build an application desktop which is later bundled with pyinstaller.

From the frontend side, I can run the bot with selenium and chromedriver.

When I use flask-socketio, for example I will run 10 processes to open 1 chrome driver each (10 chromes in total).
this is an example of interaction from the frontend to the backend flask, I click the purple play button.

![image](https://user-images.githubusercontent.com/23699888/164644945-ddf137cc-602e-47eb-81f8-28ba86306ae5.png)

when I click the purple 'play' button 1 by 1 to 10,
what happens is that only 6 chrome drivers open, chrome drivers 7 to 10 are not running. They will open when one of the first 6 chrome drivers finishes.

I didn't run using "gevent.pool" for this problem, just a plain ajax call from the frontend to the flask backend.

Please help, thank you.

**Comment by zaingithub** [comment]
![image](https://user-images.githubusercontent.com/23699888/164675960-9f0359d8-d39c-4359-8822-f6b7936fd576.png)

workaround for now, i'm using **apscheduler** to run the job with the current time.

i can run 10 chrome driver 1 by 1 and 10 chrome opens fine.

i show my simple function to run selenium.

however, I still hope there is a solution other than using apscheduler.

please help thank you very much

**Comment by miguelgrinberg** [comment]
I'm not sure if this is the problem because I don't see your implementation. Most web browsers will cap the number of WS connections that can be opened from a page. Chrome caps at 6.
